app.js

```diff
@@ -0,0 +1,9 @@
+const { toggle } = require('cli-spinners');
+
+function dropFunction() {
+  document.getElementById('drop-content').classList.toggle('show');
+}
+dropFunction();
+toggle.addEventListener('click', () => {
+  toggle.classList.dropFunction();
+});
```

Comment:
- [ ] As requested in the project's requirement, when the user clicks (or taps) any mobile menu option, a correct part of the page is displayed and the mobile menu should close. currently, it's displaying the right path, but clicking any mobile menu option is not closing the mobile menu. Kindly add this feature.